docker/pg_tests/scripts/tests/several_delta_backups_test.sh

```diff
@@ -13,21 +13,27 @@ echo "archive_timeout = 600" >> /var/lib/postgresql/10/main/postgresql.conf
 /usr/lib/postgresql/10/bin/pg_ctl -D ${PGDATA} -w start
 
 pgbench -i -s 10 postgres
+pgbench -T 100000000 postgres &
 wal-g backup-push ${PGDATA}
 
 export WALG_COMPRESSION_METHOD=lz4
-pgbench -i -s 20 postgres
+wal-g backup-push ${PGDATA}
+
+psql -f scripts/amcheck.sql postgres
```

Comment:
Even there appears ERROR it doesn't fail and return exit code 0.

Reply:
Yeah, that's a bug. Thanks, fixed.